lib/src/realm_object.dart

```diff
@@ -18,13 +18,16 @@
 
 import 'dart:async';
 import 'dart:ffi';
+import 'dart:mirrors';
```

Comment:
If I am not mistaken we can not use mirrors on Flutter

Reply:
Ugh... that's tricky. I'm using `MirrorSystem.getName` to get the member name in `noSuchMethod`. Without it, I'm not sure how we can implement it.